pangeo_forge_recipes/writers.py

```diff
@@ -95,22 +97,44 @@ def store_dataset_fragment(
 def write_combined_reference(
     reference: MultiZarrToZarr,
     full_target: FSSpecTarget,
-    output_json_fname: str,
+    concat_dims: List[str],
+    output_file_name: str,
+    output_file_type: str,
```

Comment:
Note if we infer file type from suffix we would change consolidate these into one kw.